src/lando/pushlog/management/commands/pushlog_view.py

```diff
@@ -30,9 +32,16 @@ def handle(self, *args, **options):
         except Repo.DoesNotExist:
             raise CommandError(f"Repository not found: {repo_name}")
 
-        pushes = Push.objects.filter(repo=repo).order_by("-push_id")
-        if limit:
-            pushes = pushes[:limit]
+        if push_id > 0:
```

Comment:
nit
```suggestion
        if push_id:
```

Reply:
Ah, but it's a user-controlled unsigned int. If the user passes, say, `-1`, I'd rather just show the full pushlog, rather than failing to find the non-existent push with id `-1`.

OTOH, we do have a clean way to report not-found log, so it wouldn't be a big issue 🤔